build/UIKit/dll/UIKit.def

```diff
@@ -1173,3 +1173,9 @@ LIBRARY UIKit
         UIOrientationFromString
         UIApplicationDidChangeDisplayModeNofication
         UIViewControllerFirePageEvent
+
+        ;_UIGestureCoordinator.mm 
+        _OBJC_CLASS__UIGestureCoordinator DATA
+        __objc_class_name__UIGestureCoordinator CONSTANT
+
+        
```

Comment:
You sure this is needed? It looks like you're compiling the file directly into FunctionalTests, so I don't understand why we need to export it?